packages/linux/manifest.yml

```
@@ -1,15 +1,14 @@
-format_version: 1.0.0
+format_version: 3.3.0
 name: linux
 title: Linux Metrics
 version: 0.6.12
```

Please increase the version at least to a new minor, and add a changelog entry.

Increasing it to a new minor will allow you to make backports for versions of the stack older than 8.17.0.

```suggestion
version: 0.7.0
```

If you want to avoid publishing a package with these changes by now, you may consider using the `-next` suffix.
```suggestion
version: 0.7.0-next
```